### DISCUSSION TITLE
Display of events in the backend

### DISCUSSION THREAD

**Discussion by ebussinetz** [liedekef/events-made-easy#610] [Q&A] [answered]
Is it possible to configure the display of events in the backend so that all are always shown and the newest ones first?

**Comment by liedekef** [comment]
sorry, but i use jtable for rendering the list of events and while that supports storing which columns and sizing, it doesn't support storing the sorting. So: no, not possible.

**Comment by ebussinetz** [comment]
OK, thank you. And is it possible to see all events, not only the upcoming events?

**Reply by liedekef** [reply]
you can filter on which events you want to see.

**Comment by ebussinetz** [comment]
I meant the default setting, all events

**Reply by liedekef** [reply]
No, not possible. The backend is using sensible settings that suit most people. With only 2 clicks you have the list of all events.

**Comment by ebussinetz** [comment]
But I have 166 events to edit... ;-)
The event locations and categories were not included during import and I now have to enter them...
Thank you very much for the great plugin anyway!
I had Events-List the whole time but it was no longer developed and it no longer works completely.

**Reply by liedekef** [reply, marked as answer]
Categories can be added to events in bulk, but not locations.
Just open the list of events and for editing your events open the event in another tab, so that your list stays (middle mouse click typically).

**Reply by liedekef** [reply]
Btw: https://www.e-dynamics.be/wordpress/csv-import-events-and-locations/
